test/forward_backward_compatibility/check_forward_backward_compatibility.py

```diff
@@ -98,6 +98,7 @@
     ("aten::hsplit", datetime.date(2021, 11, 20)),
     ("aten::dsplit", datetime.date(2021, 11, 20)),
     ("aten::_convolution_nogroup", datetime.date(9999, 1, 1)),
```

Comment:
Not related to this change but what this date mean?

Reply:
It's the date at which the check will begin to apply to that symbol again. Setting it to `9999` means the check will never be applied again (and this should generally not be done). The intent of this is to allow people to intentionally break the check for a short time before the check starts again.